src/code/z_demo.c

```diff
@@ -1232,683 +1401,778 @@ void Cutscene_Command_Terminator(PlayState* play, CutsceneContext* csCtx, CsCmdB
             }
         }
     }
 
-    size += 0x10;
+    size += sizeof(CutsceneCameraPoint);
 
     return size;
 }
 
-// Command 0x8: ? (Related to camera focus points)
-s32 Cutscene_Command_08(PlayState* play, CutsceneContext* csCtx, u8* cmd, u8 unused) {
-    CsCmdBase* cmdBase = (CsCmdBase*)cmd;
+s32 CutsceneCmd_SetCamAt(PlayState* play, CutsceneContext* csCtx, u8* scriptPtr, u8 unused) {
+    CsCmdGeneric* cmd = (CsCmdGeneric*)scriptPtr;
     s32 size;
-    Vec3f sp3C;
-    Vec3f sp30;
+    Vec3f at;
+    Vec3f eye;
     Camera* subCam;
-    f32 sp28;
 
-    cmd += 8;
-    size = 8;
+    scriptPtr += sizeof(CsCmdGeneric);
+    size = sizeof(CsCmdGeneric);
+
+    if ((csCtx->curFrame > cmd->startFrame) && (csCtx->curFrame < cmd->endFrame) &&
+        ((gCamAtAppliedFrame < cmd->startFrame) || (gCamAtAppliedFrame >= 0xF000))) {
+        csCtx->camAtReady = true;
+        csCtx->camLookAtPoints = (CutsceneCameraPoint*)scriptPtr;
 
-    if ((cmdBase->startFrame < csCtx->frames) && (csCtx->frames < cmdBase->endFrame) &&
-        ((D_8015FCC4 < cmdBase->startFrame) || (D_8015FCC4 >= 0xF000))) {
-        csCtx->unk_1A = 1;
-        csCtx->subCamLookAtPoints = (CutsceneCameraPoint*)cmd;
-        if (csCtx->unk_1B != 0) {
-            D_8015FCC4 = cmdBase->startFrame;
-            if (D_8015FCC8 != 0) {
+        if (csCtx->camEyeReady) {
+            // the frame number set here isn't important, it just signals that the camera data has been applied
+            gCamAtAppliedFrame = cmd->startFrame;
+
+            if (gUseCutsceneCam) {
                 subCam = Play_GetCamera(play, csCtx->subCamId);
                 subCam->player = NULL;
+
                 Play_ChangeCameraStatus(play, CAM_ID_MAIN, CAM_STAT_WAIT);
                 Play_ChangeCameraStatus(play, csCtx->subCamId, CAM_STAT_ACTIVE);
                 Play_CameraChangeSetting(play, csCtx->subCamId, CAM_SET_FREE0);
-                sp3C.x = csCtx->subCamLookAtPoints->pos.x;
-                sp3C.y = csCtx->subCamLookAtPoints->pos.y;
-                sp3C.z = csCtx->subCamLookAtPoints->pos.z;
-                sp30.x = csCtx->subCamEyePoints->pos.x;
-                sp30.y = csCtx->subCamEyePoints->pos.y;
-                sp30.z = csCtx->subCamEyePoints->pos.z;
-                Play_CameraSetAtEye(play, csCtx->subCamId, &sp3C, &sp30);
-                Play_CameraSetFov(play, csCtx->subCamId, csCtx->subCamEyePoints->viewAngle);
+
+                at.x = csCtx->camLookAtPoints->pos.x;
+                at.y = csCtx->camLookAtPoints->pos.y;
+                at.z = csCtx->camLookAtPoints->pos.z;
+
+                eye.x = csCtx->camEyePoints->pos.x;
+                eye.y = csCtx->camEyePoints->pos.y;
+                eye.z = csCtx->camEyePoints->pos.z;
+
+                Play_CameraSetAtEye(play, csCtx->subCamId, &at, &eye);
+                Play_CameraSetFov(play, csCtx->subCamId, csCtx->camEyePoints->viewAngle);
```

Comment:
you tell me, camera man PepeLaugh

Reply:
Haha, it's the same variables as `fov` so should just stick with that

Reply:
I remember having this conversation already.

conceptually, field of view **IS NOT** the same as the angle of view. Field of view represents a linear measure of what can actually be seen, whereas angle of view is a component of that calculation. 

Some reading material that backs this up:

* [Toshiba](https://www.toshiba-teli.co.jp/en/technology/technical/t0018_WhichLens.htm)

* [Matchmoving: The Invisible Art of Camera Tracking](https://books.google.com/books?id=Jr4YsaTrIBgC&pg=PA116&lpg=PA116&dq=AOV+angle+of+view&source=bl&ots=rt21Hb7DVu&sig=BWLclnvxzjbA2nciSsaM0pLaYoM&hl=en&sa=X&ei=5OT7U6OWG4K3iwLiqYDIBA&ved=0CEwQ6AEwBTgK#v=onepage&q=AOV%20angle%20of%20view&f=false)
* [The random blog I read first](https://shuttermuse.com/angle-of-view-vs-field-of-view-fov-aov/#:~:text=In%20their%20article%20it%20states,lens%20AND%20the%20sensor%20size.)

Reply:
for what its worth, debug cam strings show that they referred to this as "zoom" 
```
        DbCamera_PrintF32Bytes(point->viewAngle);
        osSyncPrintf("\t/*  zoom\t*/");
```
but yea im not knowledgeable enough in camera things to actually weigh in on fov vs angle

Reply:
(also `nextPointFrame` is "speed"... I guess it sorta makes sense cause if the next points frame is close to the start frame, thats indirectly how fast it will move to get to the end. but thats besides the point of this convo)

Reply:
Im going to consider this discussion out of scope for the PR and can be cleaned up in more camera-oriented passes